repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/nav-map.ftl

```diff
@@ -23,7 +23,7 @@
                         <#assign openInNewWindow=false/>
```

Comment:
@FranciscoRuizHEE , please remove the logic to open in new window, as discussed

Reply:
Hi @sankar-shunmuga the necessary changes have been done.